### PR TITLE
feat(IT Wallet): [SIW-362] Adds a component to parse a markdown string and get a collection of ReactNode elements

### PR DESCRIPTION
**Pull request by hevelius**
## Short description
This PR adds a new `ItwTextInfo` component that given a string of markdown text returns a set of ReactNode items using e regex expression. This component supports only regular text, bolded text (**) and link.

## How to test
Import this component in any of available screens passing ad text a markdown string ex.

`const markdownText = "Lorem ipsum dolor sit amet, **consectetur adipiscing elit**. Quisque tortor libero, **facilisis** non lectus eget, scelerisque finibus urna. [Nullam volutpat]() enim enim, sed pulvinar dolor mollis et."`

### CODE REVIEW

**Comment by dmnplb** [comment]
Hi @hevelius,
Can we use this component to format rich text in the various components that accept `ReactNode` as input? If so, it might be useful to use it throughout the application.

**Comment by hevelius** [comment]
> Hi @hevelius, Can we use this component to format rich text in the various components that accept `ReactNode` as input? If so, it might be useful to use it throughout the application.

Hi @dmnplb! This component was born precisely as a need in `it-wallet` to be able to efficiently manage texts that require inline formatting. However the nature of this component is still experimental. For now we would like to try it first in the context of `it-wallet`. However, the idea is to be able to use it later throughout the application. Please if you want to try this component. Please, if you want to try this component, it would be great to have your opinion!

File: ts/features/it-wallet/components/ItwTextInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { Text } from "react-native";
+import { Body, LabelLink } from "@pagopa/io-app-design-system";
+import { pipe } from "fp-ts/lib/function";
+import * as R from "fp-ts/lib/ReadonlyArray";
+import * as B from "fp-ts/lib/boolean";
+import { openWebUrl } from "../../../utils/url";
+import { showToast } from "../../../utils/showToast";
```
Comment (LazyAfternoons) [inline ts/features/it-wallet/components/ItwTextInfo.tsx:8]: We can replace this legacy `showToast` utility with a toast from the new DS.

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]

**Review by LazyAfternoons — APPROVED** [review]
LGTM

**Pull request merged**